src/gov/nist/javax/sip/parser/AddressParametersParser.java

```diff
@@ -64,7 +66,15 @@ protected void parse(AddressParametersHeader addressParametersHeader)
                  super.parseNameValueList(addressParametersHeader);
 
 
-            }  else super.parse(addressParametersHeader);
+            }  else {
+            	lexer.SPorHT();            	
+            	if (this.lexer.lookAhead(0) == ';') {
```

Comment:
i like this order better than the previous one, Actually, the previous commit is totally overwritten by this. Next time try to do PR with minimum commmits possible. For this kind of bug, a single commit would be ideal including code changes and junit test case